docs/sources/insight-logs/_index.md

```diff
@@ -0,0 +1,101 @@
+| `author`*             |                    Username of user who performed action.                     |
+| `author_id`*          |                       ID of user who performed action.                        |
+| `prev_state`          |                JSON representation of resource before update.                 |
+| `new_state`           |                 JSON representation of resource after update.                 |
+| `resource_id`*        |                 ID of resource on which action was performed.                 |
+| `resource_name`*      |              Name of the resource on which action was performed.              |
+| `resource_type`*      |              Type of the resource on which action was performed.              |
+| `team`*               |                    Name of team to which resource belongs.                    |
+| `team_id`             |                     ID of team to which resource belongs.                     |
+| `integration`         |                Name of integration to which resource belongs.                 |
+| `integration_id`      |                 ID of integration to which resource belongs.                  |
+| `escalation_chain`    |                    Name of team to which resource belongs.                    |
+| `escalation_chain_id` |                     ID of team to which resource belongs.                     |
+| `schedule`            |                  Name of schedule to which resource belongs.                  |
+| `schedule_id`         |                  ID of schedule to which resource belongs .                   |
+
+## Maintenance insight logs
+
+Logs are created every time when a maintenance mode is started or finished for an integration.
+
+These logs will have `action_type=maintenace` field and can be retrieved with following query:
+
+```logql
+{instance_type="oncall"} | logfmt | __error__=`` | action_type = `maintenance`
+```
+
+### Format
+
+Logs of maintenance insights contain the following fields, where the fields followed by * are always available, and the others depend on the logged event:
+
+| Field Name          |                               Description                                |
+|---------------------|:------------------------------------------------------------------------:|
+| `action_name`*      |   Name of the maintenance action, which can be `started` or `finised`.   |
+| `action_type`*      | Insight Log type. For Maintenance Insight logs it will be `maintenance`. |
+| `author`            |                  Username of user who performed action.                  |
+| `author_id`         |             Grafana OnCall ID of user who performed action.              |
+| `maintenance_mode`* |      Type of the maintenance, which can be `maintenance` or `debug`      |
+| `resource_id`*      |                  ID of integration that was maintained.                  |
```

Comment:
'was maintained' sounds weird, I tried to say "maintenance was started or finished", but shorter

Reply:
- ID of target integration.
- ID of integration under maintenance.
- ID of maintained integration.

Reply:
Please add query examples